recipes-core/images/nilrt-base-image.bb

```diff
@@ -0,0 +1,28 @@
+DESCRIPTION = "NI LinuxRT Base System Image for x64 Targets"
```

Comment:
Yeah; I was kind of hoping that I would be able to delete all the non-RAUC stuff. :)

I didn't add "RAUC" anywhere in here because it doesn't seem like a relevant part of the image's purpose. The RAUC bundle image and bundle recipe are certainly RAUC-specific, but here it is just an implementation detail. I'm hoping that it isn't going to confuse anyone for the - hopefully - short time that it coexists with the `nilrt-dkms` images (which are already labeled as "legacy" in the README.)